File: trunk/Chapter.vb
Public Class Chapter
    Private _ChapterNo As Integer
    Private _Verses As Verses

    Public Property ChapterNo() As Integer
        Get
            Return _ChapterNo
        End Get
        Set(ByVal Value As Integer)
            _ChapterNo = Value
        End Set
    End Property

    Public Property Verses() As Verses
        Get
            Return _Verses
        End Get
        Set(ByVal Value As Verses)
            _Verses = Value
        End Set
    End Property
End Class

File: Chapter.cs
namespace BibleWebService
{
    public class Chapter
    {
        private int _ChapterNo;
        private Verses _Verses;

        public int ChapterNo
        {
            get { return _ChapterNo; }
            set { _ChapterNo = value; }
        }

        public Verses Verses
        {
            get { return _Verses; }
            set { _Verses = value; }
        }
    }
}